### PR TITLE
Feature: auto connect

### PR DESCRIPTION
**Pull request by shmuel-web**
Feature: auto connect

### CODE REVIEW

File: src/stores/user.store.js [hunk 1]
```diff
@@ -65,11 +74,27 @@ class UserStore {
...
+        finally {
+            this.connecting = false
+        }
+    }
+
+    autoConnect = async () => {
+        // read from the localstorage the wallet type
+        this.walletType = window.localStorage.getItem("walletType")
+        // try to establish a connection to the previously connected wallet
+        const newConnection = false
```
Comment (yaronvel) [inline src/stores/user.store.js:91]: why do you need a new variable here? and what will happen if wallet type is null?

File: src/wallets/Wallets.js [hunk 2]
```diff
@@ -30,8 +30,8 @@ export const getMetaMask = async () => {
 
   const connectFn = async () => {
     try{
-      const accounts = await provider.request({ method: "eth_requestAccounts" })
-      return accounts[0]
+        const accounts = await provider.request({ method: "eth_requestAccounts" })
```
Comment (yaronvel) [inline src/wallets/Wallets.js:33]: please undo the extra spaces here

**Review by yaronvel — CHANGES_REQUESTED** [review]

File: src/stores/user.store.js (hunk 1 above)
Comment (shmuel-web) [reply src/stores/user.store.js]: the new variable ls for self-documenting code using verbose names improves code readability
and maintainability
I could have just passed "false" to the connect function.

for the second thing, you mentioned I fixed it

File: src/wallets/Wallets.js (hunk 2 above)
Comment (shmuel-web) [reply src/wallets/Wallets.js]: done

**Pull request merged**